packages/@react-types/tooltip/src/index.d.ts

```diff
@@ -14,35 +14,24 @@ import {AriaLabelingProps, DOMProps, StyleProps} from '@react-types/shared';
 import {OverlayTriggerProps, PositionProps} from '@react-types/overlays';
 import {ReactElement, ReactNode} from 'react';
 
-export interface TooltipTriggerProps extends OverlayTriggerProps {
+export interface TooltipTriggerProps {
   /**
    * Whether the tooltip should be disabled, independent from the trigger.
    */
   isDisabled?: boolean,
 
   /**
-   * The delay time for the tooltip to show up. [See guidelines](https://spectrum.adobe.com/page/tooltip/#Immediate-or-delayed-appearance).
-   */
-  delay?: number,
-
-  /**
-   * The additional offset applied along the main axis between the element and its
-   * anchor element.
-   * @default 7
-   */
-  offset?: number,
```

Comment:
Both `offset` and `crossOffset` are only relevant for `@react-spectrum/tooltip` as its the final UI-centric consumer. Those are also included in `PositionProps` that `SpectrumTooltipTriggerProps` extends - so there was no need to add those props anywhere in this PR.

Reply:
I think they might have been overridden for the `@default` values in the docs? @snowystinger would know.

Reply:
either way, those shouldn't be included here because this type is not related to anything that needs those 2

Reply:
True. Do the overrides need to be moved to the spectrum type then?

Reply:
Ok, I get it now - I haven't noticed that those defaults were different for `offset`. I've re-added that appropriately now.

Reply:
Verified, looks good